### ISSUE TITLE
NVDA doesn't announce cells in the selected cell range as 'selected'

### ISSUE THREAD

**Issue by AG-Zoheil** [nvaccess/nvda#18495]
**Current**
JAWS announces cells as selected, NVDA doesn't

Steps to illustrate:
1. Open https://plnkr.co/edit/50GGqPWTR8TdyMYF?open=main.js
2. Open NVDA/JAWS
3. Click a grid cell

JAWS: 
       **Actual & Expected:** cell is announced as "selected"
NVDA: 
      **Actual:** cell is not announced as "selected"
      **Expected:** cell is announced as "selected"

Inspect the DOM of the selected cell and note the aria-selected=true is set.

**Desired**
Allow NVDA to announce the aria-selected attribute set on the grid cell

**Comment by seanbudd** [comment]
Welcome @AG-Zoheil,

We are unable to process this issue as it stands.

- Please open a new issue and fill out the bug report template fully. Provide versions of software involved, including browser versions.
- To be able to reproduce this issue, we need a minimum code sample. Provide a minimal plain HTML sample to reproduce this using [codepen](https://codepen.io/). Please do not use third party frameworks.

Kind Regards,
NV Access Software Developers
